packages/matrix-client-server/src/index.test.ts

```diff
@@ -281,5 +280,77 @@ describe('Use configuration file', () => {
         ])
       })
     })
+    describe('/_matrix/client/v3/user/{userId}/account_data/{type}', () => {
```

Comment:
Do we have a test with user A accessing user B account data (and same for all other calls) 
We need tests related to (or lack of) Access Control List

Reply:
Will add this now after adding the check in the code, thank you !